deployment/ansible/group_vars/development

```diff
@@ -18,6 +18,8 @@ graphite_host: "{{ services_ip }}"
 statsite_host: "{{ services_ip }}"
```

Comment:
Unrelated to this PR, but I believe there is are two entries for `azavea.java` in `roles.txt`. Only the more recent version should be listed.